Detectors/MUON/MCH/Simulation/src/Response.cxx

```diff
@@ -39,6 +42,16 @@ Response::Response(Station station) : mStation(station)
     mSqrtK3y = 0.7642;
     mK4y = 0.38312571;
     mInversePitch = 1. / 0.25; // cm^-1
+    mChargeSlope = 10.;
+    mQspreadX = 0.18;
+    mQspreadY = 0.18;
+  }
+
+  if (mSampa) {
+    mChargeThreshold = 1e-3; //arbitrary setting for the moment, was 5*1e-4
+    mChargeSat = 3000.;      // value from aliroot for charge
```

Comment:
3000 was the saturation value for the Manu chips which had a 12-bits ADC.
The Sampa chips having only 10-bits ADCs, that value can't be right for them. Maybe Alberto | Hervé | Christophe would know which values to use here instead ?